data/types/food/carbohydrate.go

```diff
@@ -24,11 +26,12 @@ func CarbohydrateUnits() []string {
 }
 
 type Carbohydrate struct {
-	DietaryFiber *float64 `json:"dietaryFiber,omitempty" bson:"dietaryFiber,omitempty"`
-	Net          *float64 `json:"net,omitempty" bson:"net,omitempty"`
-	Sugars       *float64 `json:"sugars,omitempty" bson:"sugars,omitempty"`
-	Total        *float64 `json:"total,omitempty" bson:"total,omitempty"`
-	Units        *string  `json:"units,omitempty" bson:"units,omitempty"`
+	AbsorptionDuration *float64 `json:"absorptionDuration,omitempty" bson:"absorptionDuration,omitempty"`
```

Comment:
`absorptionDuration` should be in `Nutrition`, not as far as `Carbohydrate`. And preferably should be an int32, instead of a float64.